packetbeat/protos/http/_meta/fields.yml

```diff
@@ -10,10 +10,6 @@
           description: HTTP request
           type: group
           fields:
-            - name: params
```

Comment:
Should this become an alias type with `migration: true`? Same for the other fields which were removed.

Reply:
I thought this is what the ecs-migration's `alias: true` was going to be used for. So for any aliases that I want I should put them in fields.yml?

Reply:
Yes, you need to put them in both places.

Reply:
I added aliases for the HTTP fields where possible.